aws-sns-subscription/src/main/java/software/amazon/sns/subscription/ClientBuilder.java

```diff
@@ -4,6 +4,7 @@
 import software.amazon.cloudformation.LambdaWrapper;
 import software.amazon.awssdk.regions.Region;
 
+
```

Comment:
Make sure to add throttling here. 

https://code.amazon.com/packages/AWSCloudFormationResourceProvidersEC2VPNGateway-TB/blobs/mainline/--/vpngateway/src/main/java/software/amazon/ec2/vpngateway/ClientBuilder.java#:~:text=.build()%3B-,private%20static%20final%20RetryPolicy%20EC2_RETRY_POLICY%20%3D,.build()%3B,-%7D